cmd/root.go

```diff
@@ -77,6 +83,10 @@ func Execute() {
 		"output-format", "o", "", fmt.Sprintf(
 			"The format for the check test results.\n(Env) %s (Default) %s",
 			EnvOutputFormat, defaultOutputFormat))
+	rootCmd.Flags().StringVarP(&userCLILogFile,
+		"log-file", "l", "", fmt.Sprintf(
```

Comment:
What if we didn't give the user a choice of where the log file goes?  If we make sure users are using our container we can be prescriptive and the log location and remove a flag making the codebase simpler.  I do see the need for this if the primary user uses the Go binary directly without the container.

Reply:
As long as the default is sane, I'm good with having the option.

Reply:
> If we make sure users are using our container we can be prescriptive

Isn't this true with or without the flag?

Reply:
@tonytcampbell per our discussion offline, we'll look to solidify a single output location for the log file. I can do it in a subsequent PR.